stixcore/products/product.py

```diff
@@ -405,6 +405,55 @@ def raw(self):
     def parent(self):
         return np.unique(self.control['parent']).tolist()
 
+    def find_parent_products(self, root):
+        """
+        Conveniant way to get access to the parent products.
+
+        Performs a (inefficient) file search in the given root dir for files with the parent name.
+        Not recursive the only the direct parent is returned.
+
+        Parameters
+        ----------
+        root : `Path`
+            The fits root dir in which to find the parent file
+
+        Returns
+        -------
+        `Product`
+            A list of parent Producs (normally just one).
+        """
+
+        return [Product(f) for f in self.find_parent_files(root)]
+
+    def find_parent_files(self, root):
+        """
+        Conveniant way to get access to the parent files.
+
+        Performs a (inefficient) file search in the given root dir for files with the parent name.
+        Not recursive the only the direct parent is returned.
+
+        Parameters
+        ----------
+        root : `Path`
+            The fits root dir in which to find the parent file
+
+        Returns
+        -------
+        `Path`
+            A list of parent files pathes (normally just one).
+        """
+        if self.level == 'LB':
+            return []
+        p_level = 'LB'
+        if self.level == 'L2':
+            p_level = 'L1'
+        elif self.level == 'L1':
+            p_level = 'L0'
```

Comment:
This is highly dependent on the directory structure but I guess its not going to change. Should prob store the names in the history section